aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -47,6 +60,24 @@ public static String testBucketName() {
     return System.getenv("AWS_TEST_BUCKET");
   }
 
+  /**
+   * Get the environment variable AWS_TEST_ACCESS_POINT for a default bucket to use for testing
+   * @return bucket name
+   */
+  public static String testAccessPointName() {
```

Comment:
I think this part is not fully updated since the last comment? We don't need to get this info from environment variable, because we can just define any name we like in the test.